### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cwndrws**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# public-testing-repo
\ No newline at end of file
+# public-testing-repo
+
+Here's a line
```
Comment (duncanthedog) [inline README.md:3]: This sucks

**Review by duncanthedog — CHANGES_REQUESTED** [review]
Fix it!

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+
+Here. Is that better?
```
Comment (cwndrws) [inline README.md:5]: @duncanthedog that better?

Comment (duncanthedog) [reply README.md:5]: Yes much better

**Pull request closed without merging**